services/app/src/actions/Web.tsx

```diff
@@ -147,7 +147,11 @@ export function subscribe(a: {email: string}) {
 }
 
 export function submitUserFeedback(a: {quest: QuestState, settings: SettingsType, user: UserState, type: FeedbackType, anonymous: boolean, text: string, rating: number|null}) {
-  return (dispatch: Redux.Dispatch<any>) => {
+  return (dispatch: Redux.Dispatch<any>, getState: () => AppState) => {
+    const stats = getCounters();
+    if (stats) {
+      logMultiplayerStats(a.user, a.quest.details, stats);
```

Comment:
To clarify, it looks like these stats are getting logged separate from the users' feedback? Would it be possible to instead inject the info into their feedback, so that it's all in one place?

Reply:
Done.